### DISCUSSION TITLE
IBSng - Router OS

### DISCUSSION THREAD

**Discussion by AdrenalinK0** [eloravpn/EloraVPNManager#62] [Ideas]
اگر قابلیت اتصال به IBSng API و یا میکروتیک به پنل اضافه بشه واقعا عالیه !
چون در حال حاضر خیلیا دارن از IBSng استفاده میکنن.
از طرفی V2ray هم که چند سال اومده .
ولی هیچ پنلی که همه رو باهم پشتیبانی کنه وجود نداره.
و باید پنل های جداگانه براشون تهیه کرد.

**Comment by eloravpn** [comment]
بیشتر از چه پروتکلی استفاده می کنید؟

**Reply by AdrenalinK0** [reply]
L2TP , OPENVPN
و همه اینها رو میکرتیک هستن . با IBSNG هم کار اکانتینگ رو انجام میدم.

**Reply by AdrenalinK0** [reply]
سلام مجدد
اگر ممکن هست در مورد این موارد نظر بدین و بگین که امکان اصافه شدنشون هست یا نه .
با تشکر

**Comment by eloravpn** [comment]
سلام
با توجه به اینکه IBSNG نسخه رایگان آن دارای API نیست ظاهرا باید از ماژول زیر برای استفاده از api استفاده کرد.
https://cp.rayanagostar.com/store/webservice
بنظر فقط به صورت سفارسی قابل انجام باشه.
اگر مستنداتی برای api نسخه رایگان این نرو افزار دارید ارسال نمایید تا بررسی کنیم

**Reply by AdrenalinK0** [reply]
https://github.com/koloner/IBSNG-API-PHP/blob/main/ibsng.php
این مورد رو تو گیتهاب پیدا کردم.
